### DISCUSSION TITLE
Connection could not be established with host llc-mail.tru.io

### DISCUSSION THREAD

**Discussion by allesoverwaterstof** [JulianPrieber/littlelink-custom#333] [Issues]
Hi,

Yesterday I installed LLC, all works fines except for registering new accounts. There is an error when sending the email verification mail:

Swift_TransportException
Connection could not be established with host llc-mail.tru.io :stream_socket_client(): php_network_getaddresses: getaddrinfo for llc-mail.tru.io failed: Name or service not known

Please advice

**Comment by JulianPrieber** [comment]
This is a known issue, and we're working on fixing in it. For now, you can just ignore the error and carry on like normal.

You can remove this error by setting a custom SMTP server or disabling email verification.
